proxy/handlers/handle-sample.go

```diff
@@ -0,0 +1,21 @@
+package handlers
+
+import (
+	"context"
+
+	"github.com/byronwhitlock-google/go-mitmproxy/proxy"
+)
+
+func HandleSample(f *proxy.Flow, c CryptoClient) int {
```

Comment:
@lkolluru05 pass the interface CryptoClient defined in the handler-crypto-client.go